src/Concerns/UsesGeneratorOverrides.php

```diff
@@ -67,7 +67,7 @@ protected function viewPathUsingCanvas(string $path = ''): string
      *
      * @return array<int, string>
      */
-    protected function possibleModelsUsingCanvas(): array
+    protected function findAvailableModelsUsingCanvas(): array
```

Comment:
Add new method `findAvailableModelsUsingCanvas()` and deprecate `possibleModelsUsingCanvas()`